devtools-ci-index.yaml

```diff
@@ -681,16 +688,18 @@
             cd ..
         fi
     saasherder_deploy: |
+        echo "$KUBE_CONFIG" | base64 -d > kube_config
```

Comment:
This is how we're creating the `kube_config` file. This repeats several times in this JJB index